### PR TITLE
Task 8 - PATCH:/api/reviews/:review_id

### PR DESCRIPTION
**Pull request by omcmanus1**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -245,3 +245,107 @@ describe("POST: /api/reviews/:review_id/comments", () => {
...
+        expect(reviewObj).toBeInstanceOf(Object);
+        expect(reviewObj).toHaveProperty("review");
+        expect(reviewObj.review.length).toBe(1);
+        const expectedOutput = {
+          review_id: 2,
+          title: expect.any(String),
+          review_body: expect.any(String),
+          designer: expect.any(String),
+          review_img_url: expect.any(String),
+          votes: expect.any(Number),
```
Comment (yalicee) [inline __tests__/app.test.js:266]: have a look at what the test data votes look like and hardcode the value in to be more precise

File: __tests__/app.test.js [hunk 2]
```diff
@@ -245,3 +245,107 @@ describe("POST: /api/reviews/:review_id/comments", () => {
...
+        expect(reviewObj).toBeInstanceOf(Object);
+        expect(reviewObj).toHaveProperty("review");
+        expect(reviewObj.review.length).toBe(1);
+        const expectedOutput = {
+          review_id: 2,
+          title: expect.any(String),
+          review_body: expect.any(String),
+          designer: expect.any(String),
+          review_img_url: expect.any(String),
+          votes: -35,
```
Comment (yalicee) [inline __tests__/app.test.js:291]: like you did here???

File: __tests__/app.test.js [hunk 3]
```diff
@@ -245,3 +245,107 @@ describe("POST: /api/reviews/:review_id/comments", () => {
...
+          owner: expect.any(String),
+          created_at: expect.any(String),
+        };
+        expect(reviewObj.review[0]).toMatchObject(expectedOutput);
+      });
+  });
+  test("should respond with 201 and expected output if given extra properties", () => {
+    return request(app)
+      .patch("/api/reviews/2")
+      .send({ inc_votes: 4, faveTrick: "kickflip" })
```
Comment (yalicee) [inline __tests__/app.test.js:302]: faveTrick should be Ollie!

File: models/reviews-models.js [hunk 4]
```diff
@@ -63,3 +63,35 @@ exports.selectReviewId = (reviewId) => {
     return reviewCheck.rows;
   });
 };
+
+exports.updateReviewData = (reviewId, increment) => {
+  if (isNaN(Number(reviewId))) {
+    return Promise.reject({
+      status_code: 400,
+      message: `Invalid review ID provided`,
+    });
```
Comment (yalicee) [inline models/reviews-models.js]: PSQL error will be thrown if id is not a number please make change

**Review by yalicee — APPROVED** [review]
Very minor changes need but overall amazing code v v happy with it

File: __tests__/app.test.js (hunk 3 above)
Comment (omcmanus1) [reply __tests__/app.test.js:302]: 🤠

File: models/reviews-models.js (hunk 4 above)
Comment (omcmanus1) [reply models/reviews-models.js]: I'm a bit confused about this one - are you saying I should let PSQL do the error handling and remove checks like this?

**Pull request merged**